src/view/seed-verify.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { Header } from '../component/header';
+import { H1Text, Text } from '../component/text';
+import { Button, BackButton, GlasButton } from '../component/button';
+import { InputField } from '../component/field';
+import Card from '../component/card';
+import { FormSubText } from '../component/form';
+import { color, font } from '../component/style';
+
+//
+// Seed Verify View
+//
+
+const styles = StyleSheet.create({
+  content: {
+    justifyContent: 'flex-end',
+  },
+  title: {
+    textAlign: 'center',
+    marginBottom: 20,
+  },
+  card: {
+    maxHeight: 350,
+    maxWidth: 680,
+    paddingLeft: 45,
+    paddingRight: 45,
+    paddingBottom: 50,
+  },
+});
+
+const SeedVerifyView = ({ store }) => (
+  <Background image="purple-gradient-bg">
+    <Header>
+      <BackButton onPress={() => {}} />
+      <Button disabled onPress={() => {}} />
+    </Header>
+    <MainContent style={styles.content}>
+      <View>
+        <H1Text style={styles.title}>{"Let's double check"}</H1Text>
+      </View>
+      <Card style={styles.card}>
+        <FormSubText>{store.seedVerifyCopy}</FormSubText>
```

Comment:
I refactor the ComputedSeed to generate the complete copy and pulled the logic out of the View. This way we keep the React component as dumb as possible.

Reply:
good point